Simplified/Utilities/JWK/JWK.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright 2017 IBM Corp.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+
+import Foundation
+
+public struct JWK: Codable {
```

Comment:
can you add a comment about what kind of key(s) this can represent? I think it's for a private key but not sure if there are additional details regarding which allowed algorithms/formats etc

Reply:
JWK is a set of RSA variables, both private and public keys can be constructed with that variables. Also JWK is not limited to one specific cryptographic algorithm, so I don't  even know what to add here... we are interested in RSA data of the private key

Reply:
May be adding such a comment to our extension will be better as it is more specific? JWK structure is too general

Reply:
sure, that's fine -- I'm sort of a newbie when it comes to JWK in general, sorry if my request was unclear. I sort of remember that JWK can be used for public and private keys, and while I do see some private related fields, there were some additional fields I didn't recognize, hence my question.

Reply:
I have a link to the documentation in ASN1.swift - 
 Documentation ITU-T X.690: https://www.itu.int/rec/T-REC-X.690-201508-I/en
Added a link to RSA RFC to JWK+RSAKeys.sqift, it describes the key structure

Reply:
thanks!